policy/pa.go

```diff
@@ -130,7 +130,7 @@ var (
 	errInvalidIdentifier   = core.MalformedRequestError("Invalid identifier type")
 	errNonPublic           = core.MalformedRequestError("Name does not end in a public suffix")
 	errICANNTLD            = core.MalformedRequestError("Name is an ICANN TLD")
-	errBlacklisted         = core.MalformedRequestError("Policy forbids issuing for name")
+	errBlacklisted         = core.RejectedIdentifierError("Policy forbids issuing for name")
```

Comment:
The whole point of this pr was to prevent the user from parsing the error message ex :
```
if error.message =~ "Internationalized domain names (starting with xn--) not yet supported"
```

In our case, is was created to distinguish the blacklisted error from the others errors in this block.

Any idea?